online-shop-ui/src/components/product-list-item/product-list-item.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+import { Product } from "../../data/products";
+import "./product-list-item.scss";
+
+type ProductProps = {
+  product: Product;
+};
+
+export default function ProductListItem({ product }: ProductProps) {
+  return (
+    <>
```

Comment:
You do not need the short syntax of React.Fragment here as you've got only a <tr> and not multiple children.